crates/bevy_sprite/src/render/mod.rs

```diff
@@ -333,6 +333,27 @@ pub struct ExtractedSprites {
     pub sprites: SparseSet<Entity, ExtractedSprite>,
 }
 
+/// Allows extraction of multiply sprites for a single entity that all share that entity's id for visibility resolution.
```

Comment:
```suggestion
/// Allows the extraction of multiple sprites sharing the same entity id for visibility resolution.
```

I think there's a typo here (multiply -> multiple). I'm also leaving a suggestion to rephrase the comment, hopefully it still conveys the same meaning.